src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.ProductListUsingTagHelpers.html

```diff
@@ -58,7 +58,6 @@
     <textarea rows="4" cols="50" class="product" id="z2__Description" name="[2].Description">
 Product_2 description</textarea>
 </div>
-        
```

Comment:
I just finished investigating it. And this change is by design and is a bug fix. It's fine to get this in